db/data.js

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable indent */
+// file that will generate data and place in another file in this folder
+const fs = require('fs');
+const faker = require('faker');
+
+const RECORDS = 10;
+let STYLE_ID = 1;
+
+const createFeatures = () => {
+  const MAX = 6;
+  const MIN = 1;
+  const numOfFeatures = Math.floor(Math.random() * (MAX - MIN + 1) + MIN);
+  let fakeFeatures = '';
+  for (let i = 0; i < numOfFeatures; i += 1) {
+    const feature = faker.commerce.productMaterial();
+    const value = faker.commerce.productAdjective();
+    fakeFeatures
+      += `{
+             "feature": "${feature}",
+             "value": "${value}"
+            }`;
+    fakeFeatures += (i === numOfFeatures - 1) ? '' : ',';
+  }
+  return fakeFeatures;
+};
+
+const createPhotos = () => {
+  const MAX = 10;
+  const MIN = 6;
```

Comment:
I appreciate your making variables for this instead of just putting the numbers in the formula